src/vs/workbench/common/editor.ts

```diff
@@ -440,6 +440,7 @@ export class EditorOptions implements IEditorOptions {
 		options.forceOpen = settings.forceOpen;
 		options.revealIfVisible = settings.revealIfVisible;
 		options.pinned = settings.pinned;
+		options.forcePreview = settings.forcePreview;
```

Comment:
@wprater can you quickly check where else in this file options are handled, I think you missed to add this new flag e.g. to `TextDiffEditorOptions`